apps/survey/contracts/Survey.sol

```diff
@@ -98,84 +99,104 @@ contract Survey is AragonApp {
         return _newSurvey(_metadata, _options);
     }
 
+    /**
+     * @notice Reset previously casted vote in survey #`_surveyId`, if any.
+     * @param _surveyId Id for survey
+     */
+    function resetVote(uint256 _surveyId) public {
```

Comment:
Something I forgot about earlier; should we be using the `isInitialized` modifier for all these public functions?